CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.9.1
+[PR #93](https://github.com/fivetran/dbt_quickbooks/pull/93) includes the following updates:
```

Comment:
Should we move to this to the end of line 4, as we do with other PRs in the CHANGELOG?

Reply:
We have typically only be including the end of line PR reference when there are multiple PRs in a single CHANGELOG. When it is only one, this approach should be appropriate.